src/pybind/mgr/orchestrator/module.py

```diff
@@ -531,6 +531,17 @@ def _list_devices(self,
             out.append(table.get_string())
             return HandleCommandResult(stdout='\n'.join(out))
 
+    @_cli_write_command('orch osd zap')
+    def _zap_osd(self, osd_id: str, force: bool = False) -> HandleCommandResult:
```

Comment:
For my own information, what's the reason for zapping an osd and then not removing it? I.e., when would I use the osd zap command over the osd remove command with the zap flag set?

Reply:
@adk3798 I had thought that it would be useful in the scenario where you had removed an OSD without the zap flag and then wanted to initiate the zap manually from the orchestrator by specifying the OSD ID (vs. device paths). Now that I'm considering your question, though, I'm guessing that this wouldn't work for that scenario because we wouldn't be able to find the OSD by ID after it had already been removed. Perhaps I should remove this new orch command?